Fancyauth.API/IThing.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Fancyauth.API
+{
+    public interface IThing
```

Comment:
This interface has no meaning beyond the scope of your data structures. What you *can* do is have private wrappers. What you **can't** do is pollute the API with this stuff.

(srsly though: why not just use `object`? People can also inherit from `IThing` if they really wanna screw things up; solving this is impossible in OO langs)